microshift.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+source tools.sh
+source snc-library.sh
+
+BUNDLE_TYPE="microshift"
+INSTALL_DIR=crc-tmp-install-data
+CRC_VM_NAME=${CRC_VM_NAME:-crc}
+BASE_DOMAIN=${CRC_BASE_DOMAIN:-testing}
+MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp}
+OPENSHIFT_RELEASE_VERSION=4.12.0
```

Comment:
Is 4.12.0 still correct? This probably needs to be built from `MICROSHIFT_NVR`, or `MICROSHIFT_NVR` should be `${OPENSHIFT_RELEASE_VERSION}-${MICROSHIFT_RELEASE_SUFFIX}`?

Reply:
@cfergeau I know, what I am saying is we don't need `OPENSHIFT_RELEASE_VERSION` at line #16 and later in the script we are getting it from NVR which is good enough for this script.